packages/p2p/src/components/my-ads/my-ads.jsx

```diff
@@ -1,33 +1,48 @@
 import React, { Component, Fragment } from 'react';
 import { Button }                     from '@deriv/components';
+import Dp2pContext                    from 'Components/context/dp2p-context';
 import { localize }                   from 'Components/i18next';
+import { requestWS }                  from 'Utils/websocket';
 import FormAds                        from './form-ads.jsx';
 import { MyAdsTable }                 from './my-ads-table.jsx';
-// import ToggleAds                      from './toggle-ads.jsx';
+import ToggleAds                      from './toggle-ads.jsx';
 import './my-ads.scss';
 
 class MyAds extends Component {
     state = {
         ad_id    : '',
         // is_enabled: false,
         show_form: false,
+        is_active: 0,
+        is_loading: true,
     };
 
     handleShowForm = (show_form) => {
         this.setState({ show_form });
     };
 
+    componentDidMount() {
+        requestWS({ p2p_agent_info: 1 }).then(response => {
```

Comment:
I think it's better to use the context for this instead of making the request again and causing additional loading time

Reply:
the problem is, context won't change

Reply:
I see, there's no subscription for this? :(

Reply:
no we dont have it 😅

Reply:
can we ask 😅 ?

Reply:
let me ask